lib/routes/cookbook/index.ts

```diff
@@ -0,0 +1,150 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import logger from '@/utils/logger';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
+    path: '/',
+    categories: ['blog'],
+    description:
+        'OpenAI Cookbook 提供了大量使用 OpenAI API 的实用指南和示例代码,涵盖了从基础到高级的各种主题,包括 GPT 模型、嵌入、函数调用、微调等。这里汇集了最新的 API 功能介绍和流行的应用案例,是开发者学习和应用 OpenAI 技术的宝贵资源。',
+    maintainers: ['liyaozhong'],
+    radar: [
+        {
+            source: ['cookbook.openai.com/'],
+        },
+    ],
+    url: 'cookbook.openai.com/',
+    handler,
+};
+
+async function handler() {
+    const rootUrl = 'https://cookbook.openai.com';
+    const currentUrl = `${rootUrl}/`;
+
+    const browser = await puppeteer();
+    const page = await browser.newPage();
```

Comment:
Unnecessary use of puppeteer. The content can be fetched with `ofetch` without any issues.

Reply:
nope, I've tested and tried`ofetch`, it does not work.

Reply:
![image](https://github.com/user-attachments/assets/e492aef0-3997-4e7c-b856-efbae034bcdf)

Reply:
proxy issue...